src/Mysqldump.php

```diff
@@ -1037,7 +1040,19 @@ public function getTableLimit(string $tableName)
             return false;
         }
 
-        return is_numeric($this->tableLimits[$tableName]) ? $this->tableLimits[$tableName] : false;
+        $v = false;
+        if (is_numeric($this->tableLimits[$tableName])) {
+            $v = is_numeric($this->tableLimits[$tableName]);
```

Comment:
Should this `return $this->tableLimits[$tableName];` instead returning `true`?

Reply:
It is possible that you are right. I'll check it on Monday and change it.

Reply:
@back-2-95 I changed it and added additional tests
https://github.com/druidfi/mysqldump-php/pull/16/commits/0196b1bc986dc20dd26acb476de4ff47cdf9d8fd

Reply:
Code looks valid, I'll run the tests

Reply:
@back-2-95 https://github.com/druidfi/mysqldump-php/pull/16/commits/aeca7c0791d003934fb2ee2329a63f68f10e8116